src/collision/narrow_phase.rs

```diff
@@ -130,26 +142,35 @@ struct NarrowPhaseInitialized;
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 #[reflect(Resource)]
 pub struct NarrowPhaseConfig {
-    /// The maximum separation distance allowed for a collision to be accepted.
+    /// The default maximum [speculative margin](SpeculativeMargin) used for
+    /// [speculative collisions](dynamics::ccd#speculative-collision). This can be overridden
+    /// for individual entities with the [`SpeculativeMargin`] component.
+    ///
+    /// This is implicitly scaled by the [`PhysicsLengthUnit`].
+    ///
+    /// Default: `MAX` (unbounded)
+    pub default_speculative_margin: Scalar,
+
+    /// A contact tolerance for detecting collisions even for
+    /// slightly separated objects. This helps prevent numerical
+    /// issues and missed collisions for resting contacts.
+    ///
+    /// This is implicitly scaled by the [`PhysicsLengthUnit`].
     ///
-    /// This can be used for things like **speculative contacts** where the contacts should
-    /// include pairs of entities that *might* be in contact after constraint solving or
-    /// other positional changes.
-    pub prediction_distance: Scalar,
+    /// Default: `0.005`
+    pub contact_tolerance: Scalar,
```

Comment:
I feel like it's not clear for a user why they would want to override this and what it's effects are based on the current docs.
Also: what is the physical meaning of this number? Is it a random factor? Or a distance?

Reply:
I tried to make this a bit clearer in [c86797f](https://github.com/Jondolf/bevy_xpbd/pull/385/commits/c86797f334de1dec1a7384068bbba8fd75097161)